lkp-exec/install

```diff
@@ -388,7 +388,10 @@ distro=$_system_name_lowercase
 echo "distro=$distro"
 echo "version=$_system_version"
 
-[ "$distro" = "centos" ] && enable_centos_source
+# Enable source repositories for CentOS and Rocky Linux
+if [ "$distro" = "centos" ] || [ "$distro" = "redhat" ]; then
```

Comment:
thanks, I think redhat also need this fix. For this issue, it's better the distro name can be rocky.